autoscheduler/frontend/src/components/LandingPage/HelpText.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import {
+  Paper, Typography, Grid, Box,
+} from '@material-ui/core';
+
+const HelpText: React.SFC = function App() {
+  return (
+    <Grid
+      container
+      direction="row"
+      justify="space-around"
+      alignItems="center"
+      style={{ marginTop: '5%', marginBottom: '2%' }}
+    >
+      <Paper style={{ width: '55%', maxWidth: '800px' }}>
+        <Typography
+          variant="h3"
+          align="center"
+        >
+          <Box padding={2}>
```

Comment:
My main issue with the padding is that the top is unbalanced with the bottom. Do you think making it `padding={1} paddingTop={2}` would help with that?

Reply:
It looks like you already made some changes did you fix this?

Reply:
No, but I did now